src/client/cli/cmd/common_cli.cpp

```diff
@@ -120,10 +120,8 @@ std::string cmd::instance_action_message_for(const mp::InstanceNames& instance_n
 
 std::string cmd::update_notice(const mp::UpdateInfo& update_info)
 {
-    return ::message_box("A new Multipass version " + update_info.version() +
-                         " is available!\n"
-                         "Find out more: " +
-                         update_info.url());
+    return ::message_box(update_info.title() + " is now available!\n\n" + "Major changes and enhancements include:\n" +
+                         update_info.description() + "\n\n" + "Find out more: " + update_info.url());
```

Comment:
Don't base it off what's in the JSON today ;)

We should drive it from the JSON 1:1, otherwise we're stuck with whatever format you came up with here.

There's no reason we can't have CLI- and GUI-specific strings in there, or even translations later.

Reply:
Long-term we could even think of making these strings different depending on platform, something like:
```json
"title": {
  "default": "The default title",
  "windows-10": "Something specific to Windows",
  ...
}
```

And then query [`QSysInfo`](https://doc.qt.io/qt-5/qsysinfo.html) when displaying them.

But I don't think we need to do that just yet (OTOH we'll need a release to "enable" this before it actually works) ;)

Reply:
@Saviq,

I see the value in just using the JSON 1:1 since you say we can be in control of what it says via the website including translations.  I'll change this to reflect that.

Regarding the actual translations and platform specific strings, I say we should wait on that and work out the true requirements needed to support that.